stl/inc/algorithm

```diff
@@ -2579,14 +2579,14 @@ void _Push_heap_by_index(_RanIt _First, _Iter_diff_t<_RanIt> _Hole, _Iter_diff_t
     // percolate _Hole to _Top or where _Val belongs, using _Pred
     using _Diff = _Iter_diff_t<_RanIt>;
     for (_Diff _Idx = (_Hole - 1) >> 1; // shift for codegen
-         _Top < _Hole && _DEBUG_LT_PRED(_Pred, *(_First + _Idx), _Val); //
+         _Top < _Hole && _DEBUG_LT_PRED(_Pred, _First[_Idx], _Val); //
          _Idx = (_Hole - 1) >> 1) { // shift for codegen
         // move _Hole up to parent
-        *(_First + _Hole) = _STD move(*(_First + _Idx));
+       _First[_Hole] = _STD move(_First[_Idx]);

```

Comment:
The indentation of this line has been slightly damaged (it has been de-indented by one space), found by clang-format.